tests/factories.py

```diff
@@ -23,9 +23,14 @@
 from framework.auth.utils import impute_names_model
 from framework.sessions.model import Session
 from website.addons import base as addons_base
-from website.oauth.models import ApiOAuth2Application, ExternalAccount, ExternalProvider
+from website.oauth.models import (
+   ApiOAuth2Application,
+   ExternalAccount,
+   ExternalProvider
+)
 from website.project.model import (
-    Comment, Embargo, Node, NodeLog, Pointer, PrivateLink, RegistrationApproval, Retraction, Sanction, Tag, WatchConfig
+    Node, NodeLog, WatchConfig, Tag, Pointer, Comment, PrivateLink,
```

Comment:
What was the motivation behind reordering these imports? (from alphabetical to something more random)